libkbfs/conflict_resolver.go

```diff
@@ -910,8 +910,9 @@ func (cr *ConflictResolver) resolveMergedPaths(ctx context.Context,
 	// cache for the merged branch.
 	mergedNodeCache := newNodeCacheStandard(cr.fbo.folderBranch)
 	// Initialize the root node.  There will always be at least one
-	// unmerged path.
-	mergedNodeCache.GetOrCreate(mergedChains.mostRecentMD.data.Dir.BlockPointer,
+	// unmerged path.  Hold it here so it doesn't get gc'd.
+	rootNode, _ := mergedNodeCache.GetOrCreate(
```

Comment:
Why not make `SearchForNodes` do a `GetOrCreate` for the root node? (Not advocating for it, but just wondering if there's another reason than avoiding having to pass in the path.)